src/site/stages/build/drupal/menus.js

```diff
@@ -266,8 +266,7 @@ function makeSection(hostUrl, hub, arrayDepth, promo, pages) {
  */
 function formatHeaderData(buildOptions, contentData) {
   if (!contentData?.data?.menuLinkContentQuery?.entities) {
-    // eslint-disable-next-line no-console
-    throw new Error('formatHeaderData has no data');
```

Comment:
This was causing the build to fail with `FEATURE_ONLY_PUBLISHED_CONTENT` = `false`.
This change doesn't seem to cause any obvious harm.

Reply:
This little error sometimes serves as a catch-all for build errors that occurred earlier in the process - sometimes it means that there may have been a runtime error of some sort earlier that caused there to be no data for the MegaMenu (main navigation) in memory. Is there possibly a runtime error reported earlier on in your terminal output? Sometime between the GraphQL response finishing and up to this point